library/Fission/Challenge.hs

```diff
@@ -0,0 +1,23 @@
+module Fission.Challenge
+  ( module Fission.Challenge.Types
+  , module Fission.Challenge.Creator.Class
+  , module Fission.Challenge.Verifier.Class
+  , verificationLink
+  ) where
+
+import Fission.Challenge.Types
+import Fission.Challenge.Creator.Class
+import Fission.Challenge.Verifier.Class
+
+import Fission.Prelude hiding (link)
+import Servant         hiding (route)
+
+import qualified Fission.Web.Routes as Web 
+import qualified Fission.Web.User   as User
+
+verificationLink :: Challenge -> Text
+verificationLink challenge = 
+  toUrlPiece $ link challenge
+  where 
+    route = Proxy @(Web.UserPrefix :> User.VerifyEmailRoute)
+    link = safeLink (Proxy @Web.API) route 
```

Comment:
This could be a good helper to use in many places:

```haskell
apiLink :: forall route . Proxy route -> MkLink route Link
apiLink pxy = safeLink (Proxy @Web.API) pxy
```

Reply:
good call :+1: